RELEASE.rst

```diff
@@ -0,0 +1,27 @@
+RELEASE_TYPE: minor
+
+This release overhauls the health check system in a variety of small ways.
```

Comment:
It might be nice to linkify this:

    :doc:`the health check system <healthchecks>`